xclim/indicators/atmos/_precip.py

```diff
@@ -619,3 +620,31 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.cold_and_wet_days,
 )
+
+rain_season = PrecipWithIndexing(
+    title="Rain season",
+    identifier="rain_season",
+    realm="atmos",
+    var_name=["start", "end", "length"],
```

Comment:
```suggestion
    var_name=["rain_season_start", "rain_season_end", "rain_season_length"],
```
I think these are less ambiguous.